### PR TITLE
[Don't merge] Patch to log verbose resource usage data.

### PR DESCRIPTION
**Pull request by DmitriGekhtman**
## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: python/ray/autoscaler/_private/load_metrics.py [hunk 1]
```diff
@@ -136,6 +136,18 @@ def update(self,
...
+            logger.debug(f"Node with ip {ip} is in use.")
+            static_resources = self.static_resources_by_ip[ip]
+            dynamic_resources = self.dynamic_resources_by_ip[ip]
+            for resource in static_resources:
+                total = static_resources[resource]
+                available = dynamic_resources[resource]
+                if total != available:
+                    used = total - available
+                    logger.debug(
+                        f"Node at ip {ip} is using {used} {resource}.")
```
Comment (DmitriGekhtman) [inline python/ray/autoscaler/_private/load_metrics.py:149]: Will tell you which resources are being used.

File: python/ray/autoscaler/_private/monitor.py [hunk 2]
```diff
@@ -252,6 +252,9 @@ def update_load_metrics(self):
         response = self.gcs_node_resources_stub.GetAllResourceUsage(
             request, timeout=60)
         resources_batch_data = response.resource_usage_data
+        logger.debug(">>>Logging raw resource data.")
+        logger.debug(resources_batch_data)
```
Comment (DmitriGekhtman) [inline python/ray/autoscaler/_private/monitor.py:256]: Will tell you everything the autoscaler knows about resource usage in the cluster.

File: python/ray/autoscaler/_private/monitor.py [hunk 3]
```diff
@@ -252,6 +252,9 @@ def update_load_metrics(self):
...
+        logger.debug(">>>Done logging raw resource data.")
```
Comment (DmitriGekhtman) [inline python/ray/autoscaler/_private/monitor.py:257]: Will tell you everything the autoscaler knows about resource usage in the cluster.

**Comment by bveeramani** [comment]
## :bangbang: ACTION REQUIRED :bangbang:

We've switched our code formatter from YAPF to Black (see #21311).

To prevent issues with merging your code, here's what you'll need to do:

1. Install Black
```
pip install -I black==21.12b0
```

2. Format changed files with Black
```
curl -o format-changed.sh https://gist.githubusercontent.com/bveeramani/42ef0e9e387b755a8a735b084af976f2/raw/7631276790765d555c423b8db2b679fd957b984a/format-changed.sh
chmod +x ./format-changed.sh
./format-changed.sh
rm format-changed.sh
```

3. Commit your changes.
```
git add --all
git commit -m "Format Python code with Black"
```

4. Merge master into your branch.
```
git pull upstream master
```

5. Resolve merge conflicts (if necessary).

After running these steps, you'll have the updated `format.sh`.

**Pull request closed without merging**